packages/fxa-shared/test/guard/AdminPanelGaurd.ts

```diff
@@ -14,36 +14,37 @@ describe('support agents', () => {
   describe('Admin Panel Guard', () => {
     it('allows', () => {
       expect(
-        guard.allow(
-          AdminPanelFeature.DisableAccounts,
-          AdminPanelGroup.AdminProd
-        )
+        guard.allow(AdminPanelFeature.DisableAccount, AdminPanelGroup.AdminProd)
```

Comment:
Nit: Not related to this line, but the filename is "AdminPanelGaurd" (instead of Guard).